helm/charts/crossplane-provider-btp-cf/Chart.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v2
+appVersion: 0.0.1
+description: A Helm Chart to template crossplane manifests to manage BTP Cloud Foundry
+  resources on BTP.
+home: https://github.tools.sap/cloud-orchestration/mcp-blueprints
+icon: https://avatars.githubusercontent.com/u/45158470?s=48&v=4
+name: crossplane-provider-btp-cf
+sources:
+- https://pages.github.tools.sap/cloud-orchestration/docs/category/cloudfoundry
```

Comment:
@ricogu please remove all SAP internal links+data on every file.

Reply:
sure, there is also some SAP internal links already in some value yaml files in main branch 
example: https://github.com/openmcp-project/blueprint-building-blocks/blob/main/helm/charts/crossplane-provider-configs/values.yaml#L124

that is why I added the same in value file

Reply:
removed the links in Chart,yaml and want to check with you on if we should removed all of the link in values.yml

Reply:
removed all internal links